Configuration/PyReleaseValidation/python/relval_gpu.py

```diff
@@ -11,50 +11,59 @@
+#           Patatrack ECAL-only:              TTbar - on GPU (optional), GPU-vs-CPU validation, profiling
+#           Patatrack HCAL-only:              TTbar - on GPU (optional), GPU-vs-CPU validation, profiling
+#           Patatrack pixel-only quadruplets, ECAL, HCAL:       TTbar - on GPU (optional), GPU-vs-CPU validation, profiling
+#           Patatrack pixel-only triplets, ECAL, HCAL:          TTbar - on GPU (optional), GPU-vs-CPU validation, profiling
+#           full reco with Patatrack pixel-only quadruplets:    TTbar - on GPU (optional), GPU-vs-CPU validation, profiling
+#           full reco with Patatrack pixel-only triplets:       TTbar - on GPU (optional), GPU-vs-CPU validation, profiling
+
+# mc 2021   Patatrack pixel-only quadruplets: ZMM - on GPU (optional), GPU-vs-CPU validation, profiling
+#           Patatrack pixel-only triplets:    ZMM - on GPU (optional), GPU-vs-CPU validation, profiling
+#           Patatrack pixel-only quadruplets: TTbar - on GPU (optional), GPU-vs-CPU validation, profiling
+#           Patatrack pixel-only triplets:    TTbar - on GPU (optional), GPU-vs-CPU validation, profiling
+#           Patatrack ECAL-only:              TTbar - on GPU (optional), GPU-vs-CPU validation, profiling
+#           Patatrack HCAL-only:              TTbar - on GPU (optional), GPU-vs-CPU validation, profiling
+#           Patatrack pixel-only quadruplets, ECAL, HCAL:       TTbar - on GPU (optional), GPU-vs-CPU validation, profiling
+#           Patatrack pixel-only triplets, ECAL, HCAL:          TTbar - on GPU (optional), GPU-vs-CPU validation, profiling
+#           full reco with Patatrack pixel-only quadruplets:    TTbar - on GPU (optional), GPU-vs-CPU validation, profiling
+#           full reco with Patatrack pixel-only triplets:       TTbar - on GPU (optional), GPU-vs-CPU validation, profiling
 numWFIB = [
-           10842.502, # 10842.503, 10842.504,
-           10842.506, # 10842.507, 10842.508,
-           10824.502, # 10824.503, 10824.504,
-           10824.506, # 10824.507, 10824.508,
-           10824.512, 10824.513, # 10824.514,
-           10824.522, # 10824.523, 10824.524,
-           10824.592, # 10824.593, 10824.594,
-           10824.596, # 10824.597, 10824.598,
-           11650.502, # 11650.503, 11650.504,
-           11650.506, # 11650.507, 11650.508,
-           11634.502, # 11634.503, 11634.504,
-           11634.506, # 11634.507, 11634.508,
-           11634.512, 11634.513, # 11634.514,
-           11634.522, # 11634.523, 11634.524,
-           11634.592, # 11634.593, 11634.594,
-           11634.596, # 11634.597, 11634.598,
+           10842.502, 10842.503, 10842.504,
+           10842.506, 10842.507, 10842.508,
+           10824.502, 10824.503, 10824.504,
+           10824.506, 10824.507, 10824.508,
+           10824.512, 10824.513, 10824.514,
+           10824.522, 10824.523, 10824.524,
+           10824.582, 10824.583, # 10824.524,
```

Comment:
Out of curiosity: 10824.524 and 10824.528 here (and the same below at Ls 55-56) are just placeholders, isn't it?
To have the release numbers sequential in the tabs here, they should be 10824.584 and 10824.588 instead.
Note: I am NOT asking to modify anything here, just trying to understand the logic.

Reply:
you are right, the placeholder comments have the wrong numbers :-/